src/evaluate/visualization.py

```diff
@@ -0,0 +1,224 @@
+import textwrap
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+
+class ComplexRadar:
+    """
```

Comment:
Would you mind fixing the docstrings to match the format used in the libray: https://github.com/huggingface/evaluate/blob/dfd1cc0aea087e0876542ce8a88ab8cfa638a00c/src/evaluate/module.py#L891